### ISSUE TITLE
'sys' command produces error on Windows

### ISSUE THREAD

**Issue by jinlow** [nushell/nushell#2817]
**Describe the Bug**  
First just want to say this is an awesome project! I have really loved playing around with Nu.  However I am running into what seems like a bug on Windows.  
Running a fresh install of nushell on windows, whenever I call the `sys` command, I get the following error.
```bash
C:\Users\jinlow> sys
error: Error: Error while processing begin_filter response: Error("EOF while parsing a value", line: 1, column: 0)
```
**Steps to Repdroduce**  
Run the command `sys` in the nushell.

**Expected behavior**
Running `sys` to return a table of information about the system.

**Additional Info**
```
> version | pivot
────┬────────────────────┬──────────────────────────────────────────────────
─────────────────────────────────────────────────────────────────────────
 #  │      Column0       │                                                          Column1
────┼────────────────────┼──────────────────────────────────────────────────
─────────────────────────────────────────────────────────────────────────
  0 │ version            │ 0.24.1
  1 │ branch             │ HEAD
  2 │ short_commit       │ 5183fd25
  3 │ commit_hash        │ 5183fd25bb8e96261271740df641ff79d583a098
  4 │ commit_date        │ 2020-12-15T20:13:18+00:00
  5 │ build_os           │ windows-x86_64
  6 │ rust_version       │ rustc 1.48.0 (7eac88abb 2020-11-16)
  7 │ rust_channel       │ stable-x86_64-pc-windows-msvc (default)
  8 │ cargo_version      │ cargo 1.48.0 (65cbdd2dc 2020-10-14)
  9 │ pkg_version        │ 0.24.1
 10 │ build_time         │ 2020-12-15 20:30:18
 11 │ build_rust_channel │ release
 12 │ features           │ clipboard-cli, ctrlc, default, directories, dirs, git, ichwh, ptree, rich-benchmark, rustyline, term, trash, uuid, which,
    │                    │ zip
────┴────────────────────┴──────────────────────────────────────────────────
─────────────────────────────────────────────────────────────────────────
```
OS: Windows 10 1703 Build 15063.2108  
Nu Version: 0.24.1

**Comment by jonathandturner** [comment]
Tried this on: Version 1903 (OS Build 18362.356)
Built a debug build, and it worked. Built in release mode, and it fails the same way as above. I wonder if it has something to do with timing, as there isn't really a functional difference between debug/release.

**Comment by jinlow** [comment]
Interesting, what do you mean by timing?
So I use several different windows machines at work, and I was trying Nu on three of them. They all had this bug, but on one of them it just went away after a while... For all of them I installed Nu from *.msi on the release page.

**Comment by jonathandturner** [comment]
My hunch was how fast the code for `sys` runs on a release build vs a debug build. We use a crate that does all the system querying for us. If we're not using it correctly, or it has a bug, it's possible that bug only shows up if the queries run fast.

I don't think it has to do with installing vs not installing. I tried it locally from a freshly built nu and was able to see the same issues.

**Comment by sirxyzzy** [comment]
FWIW, I'm getting the exact same error in version  0.25.0

**Comment by schmitzli** [comment]
i guess it's fixed with 0.26.0, at least it works for me again

**Comment by jonathandturner** [comment]
We made some changes to `sys` with 0.26. I'll go ahead and close, but please reopen this if you still see issues.
